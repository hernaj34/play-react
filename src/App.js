import React, { Component } from 'react';
import './App.css';
import { Provider } from 'react-redux';
import { createStore } from 'redux';
import todoApp from './components/Todos/reducer';
import FileUploader from './components/Files/FileUploader';

let store = createStore(
  todoApp,
  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
);

class App extends Component {
  render() {
    return (
      <Provider store={store}> 
          <FileUploader />
      </Provider>
    );
  }
}

export default App;

