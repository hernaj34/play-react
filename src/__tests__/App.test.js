// Link.react.test.js
import React from 'react';
import App from '../App';
import renderer from 'react-test-renderer';

test('Application loaded', () => {
  const component = renderer.create(
    <App />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});