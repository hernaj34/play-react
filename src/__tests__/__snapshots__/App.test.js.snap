// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application loaded 1`] = `
<div>
  <h1>
    Helo world! at 
    ComponentDidMount
  </h1>
</div>
`;
